routes/ci.js

```diff
@@ -42,81 +45,178 @@ router.get('/:buildItemId/:accessToken', function(req, res, next) {
     });
 
 
-
   res.json({message: 'Request accepted'});
 });
 
 /**
- * Execute Docker.
+ * Execute Silenium plus CI Build docker.
  *
  * @param buildId
- * @param screenshotIds
- * @param newBranch
+ *  ID of CI Build on the backend.
  * @param accessToken
+ *  Access token of user creator of CI Build.
+ *
  * @returns {bluebird}
  */
 var execDocker = function(buildId, accessToken) {
-  var docker = new Docker();
-
-  var image = 'amitaibu/php-ci';
-  var cmd = [
-    '/home/shoov/main.sh',
-    buildId,
-    accessToken
-  ];
-
-  var optsc = {
-    'Env': 'BACKEND_URL=' + process.env.BACKEND_URL
+  // Init a docker.
+  var docker = new Docker({socketPath: '/var/run/docker.sock'});
+
+  // Variable contain all created containers.
+  var containers = [];
+
+  // Function create and run Silenium container.
+  var runSilenium = function() {
+    return new Promise(function(resolve, reject) {
+      var containerName = undefined;
+      var buffer = '';
+      var ready = false;
+
+      docker.createContainer({
+        'Image': 'elgalu/selenium:v2.45.0-oracle1',
+        'Env': [
+          'SCREEN_WIDTH=1920',
+          'SCREEN_HEIGHT=1080',
+          'VNC_PASSWORD=hola',
+          'WITH_GUACAMOLE=false',
+        ]
+      }, function(err, container) {
+        if (err) reject(err);
+        // Save container in containers variable.
+        containers.push(container);
+        // Attach to container.
+        container.attach({stream: true, stdout: true, stderr: true}, function(err, stream) {
+          if (err) reject(err);
+          // Start a new created container.
+          container.start(function(err) {
+            if (err) reject(err);
+            // Set timeout.
+            setTimeout(function() {
+              if (ready != true) {
+                reject("Silenium server can't start");
```

Comment:
`Silenium server couldn't start, and have timed out  after ' + timeoutLimit + ' sec'`